libp2p/Session.cpp

```diff
@@ -44,8 +44,6 @@ Session::Session(Host* _h, unique_ptr<RLPXFrameCoder>&& _io, std::shared_ptr<RLP
 {
     m_peer->m_lastDisconnect = NoDisconnect;
     m_lastReceived = m_connect = chrono::steady_clock::now();
-    DEV_GUARDED(x_info)
-        m_info.socketId = m_socket->ref().native_handle();
```

Comment:
We could output it from `admin_peers` RPC as other `PeerSessionInfo` members. Not sure if it's useful though, maybe not